pkg/apis/pipeline/v1/container_types.go

```diff
@@ -142,6 +142,15 @@ type Step struct {
 	// +optional
 	// +listType=atomic
 	Params Params `json:"params,omitempty"`
+	// Results declares StepResults produced by the step.
+	// This is field is at an alpha stability level and gated by "enable-step-actions" feature flag.
+	// It can be used in an inlined step when used to store results to
+	// $(step.results.resultName.path). When referencing StepActions,
+	// it cannot be used. The results declared by the StepActions will be stored here instead.
+	// Using it along side referencing StepActions will result in a validation error.
+	// +optional
+	// +listType=atomic
```

Comment:
Louder 😆 
```suggestion
	// Results declares StepResults produced by the step.
	//
	// This is field is at an ALPHA stability level and gated by "enable-step-actions" feature flag.
	//
	// It can be used in an inlined step when used to store results to
	// $(step.results.resultName.path). When referencing StepActions,
	// it cannot be used. The results declared by the StepActions will be stored here instead.
	// Using it along side referencing StepActions will result in a validation error.
	// +optional
	// +listType=atomic
```